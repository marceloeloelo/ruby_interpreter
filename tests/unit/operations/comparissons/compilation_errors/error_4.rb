0 >= nil # TypeError: NilClass can't be coerced into Fixnum
