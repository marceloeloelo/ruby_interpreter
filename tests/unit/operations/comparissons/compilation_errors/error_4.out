TypeError: NilClass can't be coerced into Fixnum
